content/correspondence/getting-started/developer-guides/serviceowner/_index.en.md

```diff
@@ -70,7 +77,25 @@ Use Samarbeidsportalen self-service for registration. [Here's a detailed guide](
 
 ### 7. Authentication {#authentication}
 
-For all operations, you will need to authenticate using your Maskinporten Client, then [acquire an Altinn Token from Altinn Authentication](https://docs.altinn.studio/authentication/reference/architecture/accesstoken/).
+For service owners, authentication is now more flexible with two supported methods:
+
+#### Method 1: Direct Maskinporten Authentication (New - Previously not supported)
```

Comment:
I think we can remove "(New - Previously not supported)" and "(Supported as before)" since they will become superfluous pretty quickly.